### PR TITLE
ispc:  bump llvm version needed

### PR DESCRIPTION
**Pull request by pthariensflame**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

This was missed in #100975; my apologies.

### CODE REVIEW

**Comment by pthariensflame** [comment]
I don't understand what failed here?

File: Formula/ispc.rb [hunk 1]
```diff
@@ -18,7 +19,7 @@ class Ispc < Formula
   depends_on "cmake" => :build
   depends_on "flex" => :build
   depends_on "python@3.10" => :build
-  depends_on "llvm@12"
+  depends_on "llvm@13"
```
Comment (pthariensflame) [inline Formula/ispc.rb:22]:
```suggestion
  depends_on "llvm" // Must be LLVM 13
```

**Comment by pthariensflame** [comment]
Someone help?  😅

File: Formula/ispc.rb [hunk 2]
```diff
@@ -18,7 +19,7 @@ class Ispc < Formula
   depends_on "cmake" => :build
   depends_on "flex" => :build
   depends_on "python@3.10" => :build
-  depends_on "llvm@12"
+  depends_on "llvm" // Must be LLVM 13
```
Comment (cho-m) [inline Formula/ispc.rb:22]: Need to use Ruby comment syntax.
```suggestion
  depends_on "llvm" # Must be LLVM 13
```

Comment (pthariensflame) [reply Formula/ispc.rb:22]: Whoops!  That's definitely a silly mistake; thank you!

**Review by fxcoudert — APPROVED** [review]

**Pull request closed without merging**